### PR TITLE
Game stats methods

### PR DESCRIPTION
**Pull request by thawley2**
Completed all methods for the game stats class along with their tests. 
- highest total score
- lowest total score
- percentage home wins
- percentage visitor wins
- percentage ties
- count of games by season
- average goals per game
- average goals by season

### CODE REVIEW

**Comment by CrowRising** [comment]
This was great to finish together, I think we have a great base to branch from

**Review by CrowRising — APPROVED** [review]
Great wait to finish off the Wednesday

File: lib/games_stats.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    count
+  end
+
+  def average_goals_per_game
+    sum_of_scores = @games.sum do |game|
+      game.total_score
+    end
+    sum_of_scores.fdiv(@games.length).round(2)
+  end
+### Please god refactor this method while we work###
```
Comment (andrew-bingham1) [inline lib/games_stats.rb:68]: Its majestic!

**Review by andrew-bingham1 — APPROVED** [review]
Another great session working together! Looking forward to async's too!

**Pull request merged**